hack/verify-release.sh

```diff
@@ -151,13 +150,13 @@ if [[ -n "${CONTAINER_RUNTIME}" ]]; then
     )
     declare -a OSVSCANNER_CMD=(
         "${CONTAINER_RUNTIME}" run --rm
-        -v "${PWD}":/src -w /src
-        --pull always
-        ghcr.io/google/osv-scanner:v1.9.2@sha256:239d47ec1a70af430c3cd57524d18e8b2d2dc2f28869384217b64f409ab6650a
+        -v "${PWD}":/src:z
```

Comment:
From the podman manual:

```
       Note:  Do  not relabel system files and directories. Relabeling system content might cause other confined services on the machine to fail.  For these types of containers we recommend
       disabling SELinux separation.  The option --security-opt label=disable disables SELinux separation for the container.  For example if a user wanted to volume mount their entire  home
       directory into a container, they need to disable SELinux separation.
```

Reply:
(I also don't know if :z works for docker, to be honest)

Reply:
https://docs.docker.com/engine/storage/bind-mounts/#configure-the-selinux-label it works the same.

Reply:
I'm actually surprised that any files work :) In my experiments, SELinux does not like user's home files read by containers.

Reply:
How are any of the hack scripts working for you? They all do the same, usually with `:ro,z`.